db/seeds.rb

```diff
@@ -207,6 +207,16 @@
       user.confirmed_at = Time.zone.now
     end
 
+    User.find_or_create_by!(
+      name: "Arthur",
+      email: "arthur.campbell@softwire.com",
+      organisation: org,
+      role: "support",
+    ) do |user|
+      user.password = "password"
+      user.confirmed_at = Time.zone.now
+    end
+
```

Comment:
- [] needs reverting once testing on review app complete
- [] does Jemma need a review env user to test?

Reply:
do we send emails on review? If so, can a user be created by sending an invite by a support user?